src/Uno.Extensions.Reactive/Presentation/Bindings/BindableViewModelBase.HotReload.cs

```diff
@@ -253,8 +258,8 @@ protected void __Reactive_TryPatchBindableProperties(object? previousModel, obje
 
 		if (trace) log.Trace($"Transferring state from '{previousModelType}:{previousModel.GetHashCode():X8}' to '{updatedModelType}:{updatedModel.GetHashCode():X8}'.");
 
-		var previousProperties = previousModelType.GetProperties(BindingFlags.Instance | BindingFlags.Public | BindingFlags.NonPublic).ToDictionary(prop => prop.Name, StringComparer.Ordinal);
-		var updatedProperties = updatedModelType.GetProperties(BindingFlags.Instance | BindingFlags.Public | BindingFlags.NonPublic).ToDictionary(prop => prop.Name, StringComparer.Ordinal);
+		var previousProperties = GetInstanceProperties(previousModelType);
```

Comment:
Is this required to properly enable trimming or just to get code cleaner? (Just a question)